### PR TITLE
Added support for CUDA through extension map

### PR DESCRIPTION
**Pull request by Helderjfl**
I added support to run CUDA programs.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -197,7 +197,8 @@
             ".kit": "kitc --run",
             ".v": "v run",
             ".vsh": "v run",
-            ".sass": "sass --style expanded"
+            ".sass": "sass --style expanded",
+            ".cu": "cd $dir && nvcc $fileName -o $fileNameWithoutExt && ./$fileNameWithoutExt"
```
Comment (formulahendry) [inline package.json]: On Windows, the `./$fileNameWithoutExt` would not work in CMD.

image

So we may need to use `$dir$fileNameWithoutExt`.

Please help test it on macOS, Linux and Windows. Thanks a lot!

Comment (Helderjfl) [reply package.json]: Right. I do not have a macOS to test this, I don't even know if its common, as not current Mac comes with Nvidia GPU that is mandatory for CUDA programming.
For Windows I'm trying to test it, but CUDA is not officially supported on vscode for Windows, that's why I only tried on Linux. It needs Visual Studio IDE installed to work apparently.
I'll install it and try, but you suggestions seems to work. I did not remembered that for windows its .\ and for Linux its ./

Comment (Helderjfl) [reply package.json]: Ok, just tried. It works `$dir$fileNameWithoutExt` on both Linux and Mac

**Comment by formulahendry** [comment]
Thanks @Helderjfl !

**Pull request merged**